cmd/admin-monitor.go

```diff
@@ -0,0 +1,252 @@
+/*
+ * Minio Client (C) 2019 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"sort"
+	"strings"
+
+	humanize "github.com/dustin/go-humanize"
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	json "github.com/minio/mc/pkg/colorjson"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/madmin"
+)
+
+var (
+	adminMonitorFlags = []cli.Flag{}
+)
+
+const (
+	monitor         = "BoldGreen"
+	monitorDegraded = "BoldYellow"
+	monitorFail     = "BoldRed"
+)
+
+var adminMonitorCmd = cli.Command{
+	Name:            "monitor",
+	Usage:           "monitor cpu, mem statistics",
+	Action:          mainAdminMonitor,
+	Before:          setGlobalsFromContext,
+	Flags:           append(adminMonitorFlags, globalFlags...),
+	HideHelpCommand: true,
+	CustomHelpTemplate: `Name:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} TARGET
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+  1. Get server cpu, mem statistics of the 'play' server.
+       $ {{.HelpName}} play/
+`,
+}
+
+// serverMonitorMessage holds service status info along with
+// cpu, mem, net and disk monitoristics
```

Comment:
```suggestion
// cpu and mem statistics
```

Reply:
OMG. I dont even know how it became monitoristics. Will fix